### ISSUE TITLE
Enable ignore section in browserstack.json

### ISSUE THREAD

**Issue by webunderconstruction** [browserstack/browserstack-cypress-cli#47]
My project has a sub-project with its own `node_modules` folder.

This folder is being included in the `tests.zip` file making it much larger than it needs to be, it is also making it fail once uploaded.

It would be nice to have an ignore section within the browserstack.json config to exclude files/folders.

**Comment by srebs** [comment]
Hey @webunderconstruction, interesting use case. I guess we could achieve this by adding a new CLI config flag to exclude certain files & folders.

As an immediate workaround, is it possible for you to move your tests into a subdirectory for now?

```
.node_modules
tests/
|-- cypress.json
|-- your folders & specs
```

**Comment by suryart** [comment]
This is fixed in v1.4.0.
